plugin/pkg/auth/authorizer/webhook/webhook.go

```diff
@@ -180,10 +185,10 @@ func (w *WebhookAuthorizer) Authorize(attr authorizer.Attributes) (err error) {
 		}
 	}
 	if r.Status.Allowed {
-		return nil
+		return true, "", nil
 	}
 	if r.Status.Reason != "" {
-		return errors.New(r.Status.Reason)
+		return false, r.Status.Reason, nil
```

Comment:
@ericchiang this looks wrong... if the remote hook returned `{allowed:true,reason:"because we like you"}`, the webhook authorizer would report forbidden... do you agree that just passing through the allowed and reason fields is more correct?